samples/package-lock.json

```diff
@@ -0,0 +1,6005 @@
+{
+    "name": "msal-js-samples",
```

Comment:
This package-lock was removed. Please delete and add a `.npmrc` file with the line `package-lock=false`

Reply:
Will do, thanks!